docs/config.md

```diff
@@ -267,3 +267,37 @@ spec:
     targetCacheSize: -1
     targetCacheTTL: 10
 ```
+
+
+### Authorization Properties
+
+When running on Openshift, the user is required to have sufficient permissions to certain resources that
+are mapped into Cryostat-managed resources for authorization.
+
+The mappings can be specified using a ConfigMap that is compatible with [`OpenShiftAuthManager.properties`](https://github.com/cryostatio/cryostat/blob/main/src/main/resources/io/cryostat/net/openshift/OpenShiftAuthManager.properties). For example:
```

Comment:
I think a GitHub permalink is a good idea for this or any GitHub URLs, if this file gets moved during some refactoring in the future, we could easily forget to update this documentation at the same time.

Reply:
Ahh right! I put a permalink to its current commit then:

https://github.com/cryostatio/cryostat/blob/bd95e1a11e9e29cc39559f4a5fdeaae77e81b4c6/src/main/resources/io/cryostat/net/openshift/OpenShiftAuthManager.properties

Reply:
Great!